packages/beacon-node/test/unit/eth1/utils/eth1Data.test.ts

```diff
@@ -274,3 +273,15 @@ function getMockDeposit({blockNumber, index}: {blockNumber: number; index: numbe
     depositData: {} as phase0.DepositData, // Not used
   };
 }
+
+function pick<T, K extends keyof T>(obj: T, keys: string[]): Pick<T, K> {
+  return (keys as K[]).reduce(
```

Comment:
Do we need this cast?
Also I am skeptical we need this `Pick` use here.

A different path would be to replca the `pick` usage altogether and replace it with something like:

```js
...
const eth1DatasPartial = eth1Datas.map(({blockNumber, depositCount}) => ({blockNumber, depositCount}));
...
```

Sorry for the back and forth!

Reply:
I see. 
That's must be the answer for the simplest refactoring.

Reply:
Your insights have really helped me improve my code. Thank you for your review.